Source/Scene/GlobeSurfaceTile.js

```diff
@@ -318,7 +318,7 @@ define([
 
         // If this tile's terrain and imagery are just upsampled from its parent, mark the tile as
         // upsampled only.  We won't refine a tile if its four children are upsampled only.
-        var isUpsampledOnly = defined(surfaceTile.terrainData) && surfaceTile.terrainData.wasCreatedByUpsampling();
+        var isUpsampledOnly = false;//defined(surfaceTile.terrainData) && surfaceTile.terrainData.wasCreatedByUpsampling();
```

Comment:
Sorry, I haven't seen your messages. I didn't received notifications. I was busy trying to make names working correctly.

The `isUpsampledOnly = false`  hack is only used for imagery layers not by terrain.